### PR TITLE
[FIXED] `mesheryctl system stop` bug for adapter containers removal

### LINKED ISSUE #8818: [mesheryctl] `mesheryctl system stop` command doesn't stop and remove adapter containers in docker
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

### Current Behavior

While meshery is running in docker and some mesh adapters are deployed, when `mesheryctl system stop` is run, it stops and removes meshery and watchtower containers but is unable to remove adapter containers.

### Desired Behavior

all the containers related to meshery stack running in docker must stopped by `mesheryctl system stop` command.

### Screenshots/Logs

[Screencast from 18-09-23 08:14:15 PM IST.webm](https://github.com/meshery/meshery/assets/49492007/95bf936c-6950-4a95-a3e5-08d63f58538c)

### Environment

- Host OS: Linux
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- 📺 [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- ⌨️ [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- ⌨️ [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by zakisk**
**Notes for Reviewers**

This PR fixes #8818

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by zakisk** [comment]
@leecalcote #8788 must be merged first, this PR depends on it.

**Review by Philip-21 — COMMENTED** [review]
@zakisk were you able to test these changes made locally

File: mesheryctl/internal/cli/root/system/stop.go [hunk 1]
```diff
@@ -123,25 +127,51 @@ func stop() error {
...
-		stop.Stdout = os.Stdout
-		stop.Stderr = os.Stderr
+		options := types.ContainerListOptions{
+			Filters: filters.NewArgs(
+				filters.Arg("name", "meshery"),
+				filters.Arg("network", "meshery_default"),
+				filters.Arg("label", "com.centurylinklabs.watchtower.enable=true"),
+			),
+		}
+		contaienrs, err := cli.ContainerList(context.Background(), options)
```
Comment (Philip-21) [inline mesheryctl/internal/cli/root/system/stop.go:145]: handle error calls

**Review by Philip-21 — CHANGES_REQUESTED** [review]
Please handle errors were necessary

**Comment by zakisk** [comment]
> @zakisk were you able to test these changes you made locally

Yes

**Comment by zakisk** [comment]
> @leecalcote #8789 must be merged first, this PR depends on it.

@Philip-21 you need to merge #8789 PR first then merge this PR

**Comment by leecalcote** [comment]
> @leecalcote #8789 must be merged first, this PR depends on it.

Thank you for pointing this out, @zakisk

**Comment by leecalcote** [comment]
Syncing this PR with head...

**Comment by leecalcote** [comment]
Syncing again...

**Comment by zakisk** [comment]
> Syncing again...

@leecalcote Did it work?

File: mesheryctl/internal/cli/root/system/stop.go (hunk 1 above)
Comment (zakisk) [reply mesheryctl/internal/cli/root/system/stop.go:145]: @Philip-21 do I create new error codes in [system error file](https://github.com/meshery/meshery/blob/master/mesheryctl/internal/cli/root/system/error.go)?

**Comment by saurabh100ni** [comment]
Let's discuss this on the Meshery dev call tomorrow,
Please add this as your agenda item in Docs: https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit

**Comment by zakisk** [comment]
@leecalcote @Philip-21 check this again

**Pull request closed without merging**